packages/math-input/src/components/app.js

```diff
@@ -1,16 +1,14 @@
 import {StyleSheet} from "aphrodite";
-import React from "react";
+import * as React from "react";
 
-const {View} = require("../fake-react-native-web/index.js");
-const {components, consts} = require("../index.js");
-
-const {Keypad, KeypadInput} = components;
+import {View} from "../fake-react-native-web/index.js";
+import {Keypad, KeypadInput, KeypadTypes} from "../index.js";
```

Comment:
FYI, this will require making some changes to exercises in webapp.  Flow should catch this for us though.

Reply:
It looks like we only use math-input in two places in webapp (outside of Perseus). So it should be a tiny change, and I think it's a lot cleaner than destructuring exports from `math-input`.